lib/feature_management.rb

```diff
@@ -159,4 +159,10 @@ def self.idv_by_mail_only?
       outage_status.any_phone_vendor_outage? ||
       outage_status.phone_finder_outage?
   end
+
+  def self.socure_webhook_enabled?
+    return false if Identity::Hostdata.env == 'prod'
+    return false if Identity::Hostdata.env == 'staging'
+    true
+  end
```

Comment:
Would it be more simple/conventional to just use a configuration value, and assign it in the environments we _do_ want it enabled in?

Reply:
i'm not opposed ... this approach has been previously (ie: doc auth w/ biometric comparison) used as a hard block to ensure this features cannot be enabled without a code change until we're ready 🤔

Reply:
I am a big fan of trusting our config, so we only have to write the code once, and we don't need a separate code deploy to change it later

Reply:
Yeah, I've seen the pattern before as well, though agree with @zachmargolis that we've got big problems if we can't trust our configuration. :+1: